delete/wp-site

```diff
@@ -28,18 +31,27 @@ rm -f /etc/nginx/sites-enabled/"$FQDN"
 rm -f /etc/nginx/sites-available/"$FQDN"
 rm -f /etc/php5/fpm/pool.d/"$FQDN".conf
 
-# Revoke certificate. Will try to revoke all certs. 
-for certificate in $(ls $letsencryptFolder/archive/$FQDN | grep "cert"); do
-	echo "Revoking cert $certificate"
-	../letsencrypt/letsencrypt-auto revoke --cert-path "$letsencryptFolder"/archive/"$FQDN"/"$certificate"
-	echo ""
-done
-# Remove config for domain
-rm -f "$letsencryptFolder"/"$FQDN".cli.ini
+
+# Revoke certificate if config for site exists. Will try to revoke all certs. 
+if [ -a $letsencryptFolder/$FQDN.cli.ini ]
+then
+	for certificate in $(ls $letsencryptFolder/archive/$FQDN | grep "cert"); do
+		echo "Revoking cert $certificate"
+		../letsencrypt/letsencrypt-auto revoke --cert-path "$letsencryptFolder"/archive/"$FQDN"/"$certificate"
+		echo ""
+	done
+	
+	# Remove config for domain
+	rm -f "$letsencryptFolder"/"$FQDN".cli.ini
+fi
```

Comment:
Perhaps more appropriate - prompt if there is one and ask whether user wants to revoke it or not?